lambdas/issuecredential/src/test/java/uk/gov/di/ipv/cri/kbv/api/service/fixtures/TestFixtures.java

```diff
@@ -15,11 +29,96 @@ public interface TestFixtures {
     String EC_PUBLIC_JWK_1 =
             "{\"kty\":\"EC\",\"crv\":\"P-256\",\"x\":\"E9ZzuOoqcVU4pVB9rpmTzezjyOPRlOmPGJHKi8RSlIM\",\"y\":\"KlTMZthHZUkYz5AleTQ8jff0TJiS3q2OB9L5Fw4xA04\"}";
 
+    String KBV_QUESTION_QUALITY_MAPPING =
```

Comment:
To avoid having to use the ObjectMapper outside of this class it would be useful to declare a `Map<String, int>` as well as this serialised representation of the quality mapping